Cargo.toml

```diff
@@ -7,32 +9,39 @@ members = [
 	"clap_mangen",
 ]
 
+[workspace.package]
+categories = ["command-line-interface"]
+edition = "2018"
+include = [
+    "build.rs",
+    "src/**/*",
+    "Cargo.toml",
+    "LICENSE*",
+    "README.md",
+    "benches/**/*",
+    "examples/**/*"
+]
+license = "MIT OR Apache-2.0"
+readme = "README.md"
```

Comment:
Doesn't this get made relative to the workspace root, meaning we only have one README for the entire workspace rather than per-crate READMEs?

Reply:
You are correct, that slipped my mind in doing this, I just saw "duplicate" `README.md`